### PR TITLE
Fix for https://bugs.launchpad.net/or/+bug/1956262. F5 causes OR crash.

### PR DESCRIPTION
**Pull request by mbm-OR**
More info here: http://www.elvastower.com/forums/index.php?/topic/35862-f5-causes-or-to-crash/page__view__findpost__p__280114

### CODE REVIEW

**Review by Csantucci — APPROVED** [review]
OK for me

File: Source/RunActivity/Viewer3D/Popups/TrainDrivingWindow.cs [hunk 1]
```diff
@@ -822,8 +832,13 @@ void AddLabel(ListLabel label)
...
                 else
                 {
-                    index = trainBrakeStatus.IndexOf(Viewer.Catalog.GetString("BC"));
+                    if (trainBrakeStatus.IndexOf(Viewer.Catalog.GetString("V"), index) > 0)
+                        index = trainBrakeStatus.IndexOf(Viewer.Catalog.GetString("V"), index);
+                    else
+                        index = trainBrakeStatus.IndexOf(Viewer.Catalog.GetString("BC"));
+
                     brakeInfoValue = trainBrakeStatus.Substring(index, trainBrakeStatus.Length - index).TrimEnd();
+                    brakeInfoValue = brakeInfoValue.StartsWith(Viewer.Catalog.GetString("V")) ? brakeInfoValue.Replace(Viewer.Catalog.GetString("V"), Viewer.Catalog.GetString("V") + "  ") : brakeInfoValue;
```
Comment (Sharpe49) [inline Source/RunActivity/Viewer3D/Popups/TrainDrivingWindow.cs:841]: I just have a question: what's the use of adding those spaces to the string ?

Comment (mbm-OR) [reply Source/RunActivity/Viewer3D/Popups/TrainDrivingWindow.cs:841]: The brake system tokens (**GetTrainBrakeStatus())** use two characters (BC, BP, EQ…), the "V" looks not aligned, compared to the rest, without adding those spaces to the string.

Comment (Sharpe49) [reply Source/RunActivity/Viewer3D/Popups/TrainDrivingWindow.cs:841]: Ok, thank you.

**Review by Sharpe49 — APPROVED** [review]

**Pull request merged**